### PR TITLE
Workflows/require e2e before build release

### PR DESCRIPTION
**Pull request by vinkabuki**

### CODE REVIEW

**Review by Kacper-RF — APPROVED** [review]
LGTM

File: lerna.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "version": "independent",
   "command": {
     "version": {
-      "allowBranch": ["master", "develop"],
+      "allowBranch": ["master", "develop", "workflows/require-e2e-before-build-release"
```
Comment (siepra) [inline lerna.json:8]: Leftover?

File: packages/backend/package.json [hunk 2]
```diff
@@ -83,8 +83,8 @@
 		"@chainsafe/libp2p-gossipsub": "6.1.0",
 		"@chainsafe/libp2p-noise": "11.0.0",
 		"@peculiar/webcrypto": "1.2.2",
-		"@quiet/identity": "^1.0.0",
-		"@quiet/logger": "^1.0.0",
+		"@quiet/identity": "^1.0.1-alpha.0",
```
Comment (siepra) [inline packages/backend/package.json:86]: Surely we want the alpha suffix to be here?

File: packages/identity/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@quiet/identity",
-  "version": "1.0.0",
+  "version": "1.0.1-alpha.0",
```
Comment (siepra) [inline packages/identity/package.json:3]: Same as above

File: packages/integration-tests/package.json [hunk 4]
```diff
@@ -44,8 +44,8 @@
   },
   "dependencies": {
     "@peculiar/webcrypto": "1.4.1",
-    "@quiet/identity": "^1.0.0",
-    "@quiet/logger": "^1.0.0",
+    "@quiet/identity": "^1.0.1-alpha.0",
```
Comment (siepra) [inline packages/integration-tests/package.json:47]: Same as above

File: packages/mobile/package.json [hunk 5]
```diff
@@ -22,8 +22,8 @@
   },
   "dependencies": {
     "@peculiar/webcrypto": "^1.1.7",
+    "@quiet/identity": "^1.0.1-alpha.0",
```
Comment (siepra) [inline packages/mobile/package.json:25]: Same as above

File: packages/state-manager/package.json [hunk 6]
```diff
@@ -22,7 +22,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@quiet/logger": "^1.0.0",
+    "@quiet/logger": "^1.0.1-alpha.0",
```
Comment (siepra) [inline packages/state-manager/package.json:25]: Same as above

**Pull request merged**